src/io/io_routines.f90

```diff
@@ -623,7 +625,7 @@ subroutine io_write6d(filename,varname,data_out, dimnames)
         nc=size(data_out,6)
 
         ! Open the file. NF90_CLOBBER tells netCDF we want overwrite existing files
-        call check( nf90_create(filename, NF90_CLOBBER, ncid), filename)
+        call check( nf90_create(filename, NF90_HDF5, ncid), filename)
```

Comment:
Can you "bitwise-or" this with CLOBBER so GARD doesn't break at the very end because we forgot to delete the previous file.  Or better, test to see if the file exists, and if it does add a random number to the output filename (test for file again, rinse, lather, repeat)